elixir/apps/web/lib/web/controllers/email_otp_controller.ex

```diff
@@ -256,16 +256,14 @@ defmodule Web.EmailOTPController do
       expires_at: DateTime.add(DateTime.utc_now(), session_lifetime_secs, :second)
     }
 
-    with {:ok, token} <- Auth.create_token(attrs) do
-      {:ok, Auth.encode_fragment!(token)}
```

Comment:
Is this not necessary any more because we're now encrypting the cookie so the whole cookie is encoded by default?